### DISCUSSION TITLE
Anti-Releak

### DISCUSSION THREAD

**Discussion by ghost** [HSGamer/list-of-shame-mcvn#17] [General]
`"Cracked" plugin (BlackSpigot, DirectLeaks, LeakMania,...)`
`Chính sách Anti-Releak của các trang đôi khi lại gây ảnh hưởng cho một số server (Ví dụ, Ví dụ 2,...)`
Don't want to be banned by our anti-releak, don't releak.
It's simple as that.

**Comment by ghost** [comment]
Sites like DirectLeaks, LeakMania have Anti-Releak just because of people reposting/releaking resources on some other random sites, That is against rules on all sites, so they have Anti-Releak to combat that.

**Comment by HSGamer** [comment]
All of them are temporary solutions for using premium plugins. I won't recommend using leak plugins anyway.
I was in a period when Anti-Releak in some sites was buggy.
Better be aware.

**Comment by HSGamer** [comment]
> It is only buggy if you try to bypass it.

You may be correct.
One other case is that you don't update the plugins after some times. Could be the Anti-Releak system being updated.

**Comment by HSGamer** [comment]
> Btw, you say that LM is a shame but you using it:
> ![1](https://user-images.githubusercontent.com/48015201/115996902-6d9a8580-a5e1-11eb-996b-e8587faedc73.png)

Used to. Why should I not login back to see my account after a long time? If you want, you can delete my account.

**Comment by HSGamer** [comment]
> The Anti-Releak system is completely separated the the plugin update system.
> When we found a update system that disable the plugin if it is not up-to-date, we removing the update system.
> You will only find that problem on BSMC because their staff has no cracking skills.

Those "downloaders from releakers" don't know what they are doing and still ask us why their servers shut down.

BSMC is the one to say. I hate that site.

This is for a community with those newbie Vietnamese people make their servers with no skills and little knowledge on what they are doing.

Like I said, "A temporary solution but risky in long-term".

You have your reason for Anti-Releak, and you care about your users. I appreciate that. But I don't know other sites, besides you (and possibly DL), are doing what you are doing.

**Comment by ghost** [comment]


**Reply by HSGamer** [reply]
> The server is shutting down because you downloaded a releaked version of the plugin.
> If you download it directly from LM and you not share the file anywhere, you will not have this problem

This should be enough for a disclaimer in the list

> I not specifically hate BSMC but the staff are skids.

Agree

> So, instead of releaking the file on MCVN, add the LM link to the resource to allow them to directly download the files on LM.

As the current owner of MCVN forum is inactive, and I (and possibly others) have the MOD badge but no privilege when dealing with those releakers (can't delete those resources), the MCVN rules are useless in that section. Until the owner does something to revamp the forum, this will still be a problem between the leaking sites and MCVN releakers.

> The goal of LM is only to allow our members to test premium plugins before buying them because on majors premium plugins there is no trial versions.
> We never said to our members to use them in production (it's possible, but it's better to puchase the plugin to support the plugin author).

Some people are greedy and use cracked plugins in their production servers. That's a problem in the MCVN community.

**Comment by ghost** [comment]


**Reply by justmangoou** [reply]
It's very difficult to get everyone at MCVN to understand the reasons why should not re-leak plugins from BSMC or LeakMania or directleaks, etc... As far as I know that the owner of MCVN's site don't really care about this problem.

**Reply by HSGamer** [reply]
I think not. If they understood the ToS, they would know the consequence of releaking. 
Maybe they skipped it and checked the Agree box.
It's weird that checking the box doesn't mean that they accepted the ToS.

**Reply by justmangoou** [reply]
Accepting ToS is just a confirmation that they will have responsibility for their actions.
But it won't prevent re-leakers from publishing resources from your website to MCVN.
